python_ta/contracts/__init__.py

```diff
@@ -176,6 +180,15 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
     return r
 
 
+def _get_argument_suggestions(arg: Any, annotation: type) -> str:
+    """Returns potential suggestions for the given arg and its annotation"""
+    suggestions = ""
```

Comment:
I don't think we're going to accumulate suggestion texts very much, so I prefer just using an `if ... else ...` structure for this function.